templates/security/index.html

```diff
@@ -42,15 +42,15 @@ <h2>Find out more</h2>
   </div>
   <div class="row u-equal-height">
     <div class="col-4 p-card u-align--center">
-      <p><a href="https://insights.ubuntu.com/tag/security"><img src="{{ ASSET_SERVER_URL }}df54bb6a-document-open-icon.svg" width="56" alt="" style="max-height: 47px;" /></a></p>
+      <p><a href="/blog/tag/security"><img src="{{ ASSET_SERVER_URL }}df54bb6a-document-open-icon.svg" width="56" alt="" style="max-height: 47px;" /></a></p>
```

Comment:
It was already like this, but should this icon link to a different URL than the text below it? The icon points to `/blog/tag/security` and the text to `https://usn.ubuntu.com/usn/`